demisto_sdk/commands/validate/file_validator.py

```diff
@@ -339,6 +340,11 @@ def validate_modified_files(self, modified_files):  # noqa: C901
                 if self.is_backward_check and not incident_field_validator.is_backward_compatible():
                     self._is_valid = False
 
+            elif checked_type(file_path, [REPUTATION_REGEX]):
```

Comment:
should also include `Packs` path option?

Reply:
No. indicators are only in Misc repo